hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/ClientCacheProxy.java

```diff
@@ -340,6 +342,30 @@ public void registerCacheEntryListener(CacheEntryListenerConfiguration<K, V> cac
         }
     }
 
+    @Override
+    public void registerCacheEntryListener(CacheEntryListenerConfiguration cacheEntryListenerConfiguration, boolean addToConfig) {
```

Comment:
You could deduplicate code and implement `registerCacheEntryListener(CacheEntryListenerConfiguration<K, V> cacheEntryListenerConfiguration)` by invoking  `registerCacheEntryListener(cacheEntryListenerConfiguration, true)`.